core/quorum_recv.go

```diff
@@ -1280,8 +1292,9 @@ func (c *Core) getProofverificationDetails(tokenID string, senderAddr string) (s
 			return "", "", err
 		}
 		//check if token chain of token pledged for already synced to node
+		//TODO: Change proof verification method
 		pledgedfBlk, err := c.w.GetTokenBlock(tokenPledgedFor, tokenPledgedForType, tokenPledgedForBlockId)
-		if err != nil {
+		if err != nil && err.Error() != "leveldb: not found" {
```

Comment:
Please add a comment here explaining what the if condition is doing

Reply:
Condition changed